spec/controllers/invitations_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require "spec_helper"
+
+describe InvitationsController do
+  describe "GET new" do
+    it "sets the @invitation to a new invitation" do
+      set_current_user
+      get :new
+      expect(assigns(:invitation)).to be_new_record
```

Comment:
The reason to do that would be to combine these assertions, just in case I wasn't clear.